server/src/main/java/io/crate/planner/operators/JoinPlan.java

```diff
@@ -77,6 +92,22 @@ public boolean isRewriteFilterOnOuterJoinToInnerJoinDone() {
         return rewriteFilterOnOuterJoinToInnerJoinDone;
     }
 
+    public boolean moveConstantJoinConditionRuleApplied() {
+        return moveConstantJoinConditionRuleApplied;
+    }
+
+    public JoinPlan withMoveConstantJoinConditionRuleApplied(boolean moveConstantJoinConditionRuleApplied) {
```

Comment:
Only thing is, we don't use `false`.

Reply:
This is true, but it makes it very explicit what is going on.